src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -740,6 +741,70 @@ std::set<std::vector<element::Type>> jit_gelu_tanh_emitter::get_supported_precis
     return {{element::f32}};
 }
 
+/// GELU_V7 ///
+jit_gelu_v7_emitter::jit_gelu_v7_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                         dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                         const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    const auto gelu = std::dynamic_pointer_cast<ov::op::v7::Gelu>(node);
```

Comment:
Minor: please use references to avoid redundant copies. Here and in the SWISH emitter on L1686.
```suggestion
    const auto& gelu = std::dynamic_pointer_cast<ov::op::v7::Gelu>(node);
```

Reply:
Applied. Thanks Ivan!